flights/berks_ceh_clients_flight.yaml

```diff
@@ -94,13 +92,17 @@ entityDefinitions:
         transforms:
         - !<transforms.ReplaceTransform>
           target: ["0", "1", "8", "9", "99"]
-          goal: ["Nonhispanic", "Hispanic", "Client doesn't know", "Client refused", "Data not collected"]
+          goal: ["nonhispanic", "hispanic", "", "", ""]
           partial: true
       ol.datasource:
         type: "ol.datasource"
         transforms:
         - !<transforms.ValueTransform>
           value: "Berks County CEH"
+    conditions:
+    - !<conditions.BooleanIsNullCondition>
```

Comment:
Whenever we add the datasource, we need to add a condition to the person entity set, otherwise too many entities get written in. This is because even if there is no data, a row will be written because the datasource comprises one data point. So you sometimes get a lot of unwanted blank rows. If there are rows without a PersonalID that you still want to be read in as a person then add conditions here about what the cutoff is for not writing a person.